ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3614,7 +3614,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
-- as: vsphere-ipi-proxy-fips-f28
+- as: vsphere-ipi-proxy-fips-regenerate-cluster-cert-f28
```

Comment:
The test names are limited to 61 chars. Perhaps rename to `recert` so that it can be applied for the other complicated profiles which has long names.

Reply:
I was thinking about the name too, but not from the perspective of length. Instead, was thinking about keeping consistency as past. In past, many people call `"cert-rotation"` to mean test name, chain name, script name ...

Given Liang Xia issues a comment, I'd suggest still use well-known `"cert-rotation"` instead of `"regenerate-cluster-cert"`.
There is a totally new tool https://github.com/rh-ecosystem-edge/recert/blob/main/docs/design.md we are asked to cover with https://github.com/openshift/release/pull/45637 . So we'd better to not use `"recert"` wording for past `"cert-rotation"`.

Reply:
> And I found anothrer cert-rotation in openshift e2e test, so I don't suggest we use cert-rotation as name. In order to distinguish,

For this point , on the contrary, I think we should keep consistent with dev's ci jobs, so that easily find out similar test coverage, whatever from QE or from Dev team.

Reply:
@jianlinliu That's a different chain, it did a test following this doc https://cloud.redhat.com/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms, current pr we did following this https://access.redhat.com/articles/regenerating_cluster_certificates based on fips on cluster, quite different.

Reply:
If that, deserve a different name.

Reply:
Then how about Dev's `"certrotation"` `https://github.com/openshift/release/pull/40388`?
```
$ tree ci-operator/step-registry/openshift/oc/certrotation
ci-operator/step-registry/openshift/oc/certrotation
├── openshift-oc-certrotation-commands.sh
├── openshift-oc-certrotation-ref.metadata.json
├── openshift-oc-certrotation-ref.yaml
```

Reply:
`regen-cluster-cert` is still a bit long, if we extend the testing on more install workflow, the base install install workflow is already long, so it is better to have short name as much as possible

Reply:
@xingxingxia That's from different Devs, that's not our problem at the moment.
@jianlinliu Okay, What about using `regen-cert` as name?

Reply:
> What about using regen-cert as name

okay for me.

Reply:
`"certrotation"` `https://github.com/openshift/release/pull/40388` is not from different Devs, it is exactly our Devs for the June FIPS test period, e.g. David Eads `https://github.com/openshift/release/pull/40388#issuecomment-1594657083`, our QE's script is based on `https://github.com/openshift/release/pull/40388/files`'s script

Reply:
regen-cert LGTM too.

Reply:
Discussed with xingxing, will use `regen-cert` and add related Docs link to the chain description.